src/pybind/mgr/dashboard/frontend/src/app/shared/components/critical-confirmation-modal/critical-confirmation-modal.component.ts

```diff
@@ -45,6 +52,36 @@ export class CriticalConfirmationModalComponent implements OnInit {
     }
   }
 
+  git clone --depth 1 --single-branch git@github.com:ceph/ceph.git
+  git clone --depth 1 --single-branch https://github.com/himanshu-0611/ceph-dev.git
```

Comment:
we can remove these lines
`git clone --depth 1 --single-branch git@github.com:ceph/ceph.git
  git clone --depth 1 --single-branch https://github.com/himanshu-0611/ceph-dev.git`